cypress/support/step-definitions/grid-steps.js

```diff
@@ -38,6 +44,16 @@ Then('pod {int} has width from column {int} to column {int}', (index, colStart,
     .and('have.css', 'grid-column-end', `${colEnd}`);
 });
 
+Then('justify pod {int} has style for justify {word}', (index, justify) => {
+  justifyGridComponent().children('div').children('div').eq(index)
+    .should('have.css', 'justify-self', `${justify}`);
+});
+
+Then('align pod {int} has style for align {word}', (index, align) => {
+  alignGridComponent().children('div').children('div').eq(index)
```

Comment:
the same should be here: 
replace `index` with `position` and use `positionOfElement(position)` please

Reply:
fixed